manifests/role/hathitrust/dev_and_ruby.pp

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# Buzz
```

Comment:
I'm fine with this, but buzz is essentially a HT dev app host if we want to go that route now.

Reply:
Done, but might cause problems. I haven't looked at the named_instances profile yet to see if adding it to buzz would be a problem.

Reply:
Update: we looked through it, it looks ok.